src/main/java/hudson/security/LDAPSecurityRealm.java

```diff
@@ -413,6 +414,9 @@ group target (CN is a reasonable default)
 
     private boolean disableRolePrefixing;
 
+    @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "Diagnostic fields are left mutable so that groovy console can be used to dynamically turn/off probes.")
+    //public static boolean getCnFromAttributes = Boolean.getBoolean(LDAPSecurityRealm.class.getName()+".getcnfromattributes");
```

Comment:
and since only line 418 is commented; `@SuppressFBWarnings` is now applied to the constructor below

Reply:
I'm removing it because the main proposed change is assigning the groupName from the fist CN value instead of doing it from last string value of the dn....why? it is not mandatory to have `cn` included into the `dn` string but it is mandatory to have defined a `cn` attribute for any group record of the LDAP schema